libr/core/project.c

```diff
@@ -648,11 +648,20 @@ static bool projectSaveScript(RCore *core, const char *file, int opts) {
 		ohl = strdup (hl);
 		r_cons_highlight (NULL);
 	}
-
 	fdold = r_cons_singleton ()->fdout;
 	r_cons_singleton ()->fdout = fd;
 	r_cons_singleton ()->context->is_interactive = false;
 	r_str_write (fd, "# r2 rdb project file\n");
+	if (!core->bin->is_debugger && !r_config_get_i (core->config, "asm.emu")) {
+		char *fpath = r_file_abspath (core->bin->file);
+		if (fpath != NULL){
```

Comment:
!= NULL is implicit. just `if (fpath) {`. addd proper spacings and indent the block below